src/semantic_hashing_demo/generate_data.py

```diff
@@ -0,0 +1,60 @@
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import marvin
+import polars as pl
+from pydantic import BaseModel, Field
+
+
+class ParagraphData(BaseModel):
+    original: str = Field(description="The original paragraph")
+    very_similar: str = Field(
+        description="A paragraph that is almost identical to the original paragraph with only a couple of words changed"
+    )
+    # semantically_similar:str
+
+
+# data = []
+# for _ in range(5):
+#     data.extend(marvin.generate(n=3, target=ParagraphData, instructions="generate paragraphs for comparison testing. the paragraphs should be almost identical, with only a few words changed. each paragraph should be at least 100 words long."))
+#     print(data)
+# data_dicts = [d.dict() for d in data]
+# df = pl.DataFrame(data_dicts)
+# df.write_csv("./data/paragraphs.csv")
```

Comment:
This can be deleted.

Reply:
[Commit](https://github.com/abhi3700/semantic-hashing-demo/pull/2/commits/506a8c9ea571e9706b10a1b80b2ac0ff8959dd61).